docs/source/config-language.rst

```diff
@@ -0,0 +1 @@
+.. mdinclude:: config-language.md
```

Comment:
When I run Sphinx it is complaining:

```
WARNING: multiple files found for the document "config-language": ['config-language.md', 'config-language.rst']
Use '/home/ubuntu/Sinergise/eo-grow/docs/source/config-language.rst' for the build.
```

I believe that you don't need the rst file and can only keep the markdown.

Reply:
yes, still works, thanks :D